sjb/config/test_cases/test_pull_request_origin_extended_image_registry.yml

```diff
@@ -11,4 +11,20 @@ extensions:
       repository: "origin"
       script: |-
         OS_BUILD_ENV_PULL_IMAGE=true OS_BUILD_ENV_PRESERVE=_output/local/bin/linux/amd64/extended.test hack/env make build-extended-test
-        OPENSHIFT_SKIP_BUILD='true' KUBECONFIG=/etc/origin/master/admin.kubeconfig TEST_ONLY=true JUNIT_REPORT='true' make test-extended SUITE=core FOCUS="\[Feature:Image|\[registry\]"
\ No newline at end of file
+
+        export OPENSHIFT_SKIP_BUILD=true
+        export KUBECONFIG=/etc/origin/master/admin.kubeconfig
+        export TEST_ONLY=true
+        export JUNIT_REPORT=true
+
+        source "hack/lib/init.sh"
```

Comment:
I think we will fail to detect `$OS_ROOT` correctly since the script that runs is not under the Origin tree. I may have made that work in the past but I can't remember. You should test that this script, when saved in some part of the filesystem NOT in the Origin tree, works as expected.

Reply:
Can I at least assume that `$OS_ROOT` is set?

Reply:
No, nothing sets it

Reply:
@miminar i'm holding merging/creating this based on @stevekuznetsov's response, but if i'm wrong let's talk tomorrow and i can set it up for you and we can see what happens.

Reply:
So the `"$(pwd)"` holds the truth. I've made `"${OS_ROOT}"` guessing a bit more robust in this script just in case...